move/operators/Move.toml

```diff
@@ -4,7 +4,7 @@ version = "0.1.0"
 edition = "2024.beta"
 
 [dependencies]
-Sui = { git = "https://github.com/MystenLabs/sui.git", subdir = "crates/sui-framework/packages/sui-framework", rev = "mainnet-v1.25.3" }
```

Comment:
update its/squid as well. just don't need to format them

Reply:
updated